script/download-ffi.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -eu
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)" # Figure out where the script is running
 . "$SCRIPT_DIR"/ci/lib/robust-bash.sh
-
+. "$SCRIPT_DIR"/ci/lib/utils.sh
```

Comment:
Could we call this something that isn't `utils`? In this case, it looks like `download-file.sh` might be alright

Reply:
Why did I know you'd say that :P